README.md

```diff
@@ -14,6 +14,26 @@ What's this new konnector?
 The konnector gets information from Malakoff Mederic (french national insurance)
 This konnector fetches the list of reimbursements from [Malakoff Mederic][malakoff_mederic].
 
+What data is imported ?
+-----------------------
+
+This konnector imports the list of reimbursements metadata that you can see in the "Suivi des remboursements" page. It uses the detailed view for each line.
```

Comment:
I would start by stating that it fetches io.cozy.bills (it is implied below since the documentation is linked to io.cozy.billsdocumentation). What is the Suivi des remboursements page ? If I do not have Santé, I don't know what it means. What does "detailed view" for each line mean ?

Reply:
Maybe under each data is imported we could have the same view that is shown in collect when adding the konnector ? It would allow use to templatize the README.md.

Reply:
"Suivi des remboursement" is a reference to a page on Malakoff Mederic's page. That is why I keep it in french. I will make it clearer